src/App.tsx

```diff
@@ -17,18 +82,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                selectStatus={selectStatus}
+                query={query}
+                onSelectStatus={handleSelectStatus}
+                onQueryChange={handleQueryChange}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos ? (
+                <Loader />
+              ) : (
+                <TodoList
+                  todos={todos}
+                  selectedId={selectedTodo?.id || 0}
```

Comment:
make this field optional
```suggestion
                  selectedId={selectedTodo?.id}
```